### DISCUSSION TITLE
Problem with loading product Images in new Theme

### DISCUSSION THREAD

**Discussion by Andrew-SE** [bagisto/bagisto#10420] [General]
Hello
I have problem with loading images on store front, ***"The image ... cannot be displayed because it containes errors."*** On local env, running ```php artisan serve```, Bagisto 2.2
I have tried all the solution to similar problems, however none worked.

* Re: [Product Images Not Showing due to cache issues](/topic/3501/product-images-not-showing-due-to-cache-issues)

* Re: [ Plz help](https://forums.bagisto.com/topic/3139/plz-help-why-all-images-with-product-images-are-not-showing-like-this?_=1734643774019)
* Re: [ images no showing in the front store error. This is the solution. Tested and works](https://forums.bagisto.com/topic/1367/images-no-showing-in-the-front-store-error-this-is-the-solution-tested-and-works/1?_=1734643774022)

I have reseted the imagecache.php to default, switched channel to default, unlinked, linked, cleared cache, changed url. Nothing worked, even in default channel

```
APP_NAME=Bagisto
APP_ENV=local
APP_KEY=base64:/7WmXWFG1G7BxiXdJoSA4a7DJZf/5ebkzAn6Bx80QJ4=
APP_DEBUG=true
APP_DEBUG_ALLOWED_IPS=
APP_URL=http://localhost:8000
APP_ADMIN_URL=admin
APP_TIMEZONE=Europe/Prague
APP_LOCALE=en
APP_CURRENCY=USD

VITE_HOST=
VITE_PORT=

LOG_CHANNEL=stack

DB_CONNECTION=mysql
DB_HOST=127.0.0.1
DB_PORT=3306
DB_DATABASE=sourbelts
DB_USERNAME=root
DB_PASSWORD=200211
DB_PREFIX=

BROADCAST_DRIVER=log
CACHE_DRIVER=file
SESSION_DRIVER=file
SESSION_LIFETIME=120
QUEUE_DRIVER=sync

REDIS_HOST=127.0.0.1
REDIS_PASSWORD=null
REDIS_PORT=6379

RESPONSE_CACHE_ENABLED=false

MAIL_MAILER=smtp
MAIL_HOST=smtp.mailtrap.io
MAIL_PORT=2525
MAIL_USERNAME=
MAIL_PASSWORD=
MAIL_ENCRYPTION=tls

MAIL_FROM_ADDRESS=shop@example.com
MAIL_FROM_NAME=

ADMIN_MAIL_ADDRESS=admin@example.com
ADMIN_MAIL_NAME=

FIXER_API_KEY=
EXCHANGE_RATES_API_KEY=
EXCHANGE_RATES_API_ENDPOINT=

PUSHER_APP_ID=
PUSHER_APP_KEY=
PUSHER_APP_SECRET=
PUSHER_APP_CLUSTER=mt1

MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"

FACEBOOK_CLIENT_ID=
FACEBOOK_CLIENT_SECRET=
FACEBOOK_CALLBACK_URL=https://yourhost.com/customer/social-login/facebook/callback

TWITTER_CLIENT_ID=
TWITTER_CLIENT_SECRET=
TWITTER_CALLBACK_URL=https://yourhost.com/customer/social-login/twitter/callback

GOOGLE_CLIENT_ID=
GOOGLE_CLIENT_SECRET=
GOOGLE_CALLBACK_URL=https://yourhost.com/customer/social-login/google/callback

LINKEDIN_CLIENT_ID=
LINKEDIN_CLIENT_SECRET=
LINKEDIN_CALLBACK_URL=https://yourhost.com/customer/social-login/linkedin-openid/callback

GITHUB_CLIENT_ID=
GITHUB_CLIENT_SECRET=
GITHUB_CALLBACK_URL=https://yourhost.com/customer/social-login/github/callback

```

Any help is welcome.  :construction_worker: 
Thanks for reading

**Comment by devansh-webkul** [comment]
Hello there,

Kindly ensure that the GD extension is enabled. You can refer to the requirements listed here:
https://devdocs.bagisto.com/2.2/introduction/requirements.html#php-extensions

**Comment by Andrew-SE** [comment]
GD enabled

**Comment by jackhenry123s** [comment]
i think Gt issues (GT Enabled)

**Comment by Andrew-SE** [comment]
I've tried trouble shooting.  My notes:
- On new project everything work perfectly.
- Created Custom package 'fooPackage' for frontend (_copy of shop package_), all namespaces `MyPackages\fooPackage` it worked
- however at some point  It just stopped working. I think there might be some edge case with `imagecache` and `CacheFilters` after  re - publishing package (`vendor:publish --force`), and sometimes i starts displaying it again

- Images imported in code work all the time ` <img
        src="{{ bagisto_asset('/images/foo.png') }}"
        class="..."
        alt="foo">`
- images only that are computed with `imagecache` are bugged

Didn't have time to dig deeper yet

**Comment by jackhenry123s** [comment]
Check if image paths are correct and publicly accessible in public/storage. Also, try running php artisan storage:link again and clear all caches. [chipotle promo codes ](https://restocouponcodes.com/chipotle/)
